Tests/TestsCommon/CommonTestUtils.swift

```diff
@@ -0,0 +1,172 @@
+import Foundation
+import mongoc
+@testable import MongoSwift
+import Nimble
+import XCTest
+
+// sourcery: disableTests
```

Comment:
does this mean none of the tests are currently running?

Reply:
nah, this is so that sourcery doesn't include the base class in the list of generated tests in LinuxMain, see [here](https://github.com/mongodb/mongo-swift-driver/blob/1cd83228be80c121c6eb0670f77eab8477de7182/Tests/LinuxMain.stencil#L5)